packages/nx-semantic-release/src/config/apply-tokens.spec.ts

```diff
@@ -28,6 +28,7 @@ describe('applyTokensToSemanticReleaseOptions', () => {
       commitMessage: 'release ${PROJECT_NAME} in ${PROJECT_DIR}',
       packageJsonDir: '${PROJECT_DIR}/src',
       tagFormat: '${PROJECT_NAME}-v${version}',
+      outputPath: '${PROJECT_DIR}/../../dist/apps/${PROJECT_NAME}'
```

Comment:
It will probably work in most projects, but I think that we should mention that this is the default value in readme :)

Reply:
Humm...not sure this should be mentioned as default value. the base folder  may vary:  `apps`, `libs` or `packages`

Reply:
Instead, what we can do (in another PR probably), is to infer the `outputPath`, from the `buildTarget` executor.
In fact, the info is available there as well. E.g:

```
  "targets": {
    "build": {
      "executor": "@nrwl/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/packages/nx-micronaut",
        "main": "packages/nx-micronaut/src/index.ts",
        "packageJson": "packages/nx-micronaut/package.json",
        "tsConfig": "packages/nx-micronaut/tsconfig.lib.json",
        "updateBuildableProjectDepsInPackageJson": true,
        "buildableProjectDepsInPackageJsonType": "dependencies",
        "assets": [

        ]
      }
    },

Reply:
Haha, I just realized that this is the test file, I thought it would be the default value used in the plugin itself 😁.
That happens when you do the code review in the evening.

Anyway, I like the idea of inferring `outputPath` from `buildTarget`! I'll introduce that in the next release.